v3/integration/lints/utils/filters.go

```diff
@@ -0,0 +1,33 @@
+package utils
```

Comment:
I know ZLint already has a `utils` package already but I've been trained to think of this as a small code smell.

The `filters.go` name suggests that maybe we could call this package `filters` instead, or maybe `fileops` or something that hints it's common functions for handling types of files. "Utils" is an invitation to throw in anything :-)

Reply:
Right!? I struggled with this because it seemed super `utils`-y and I just couldn't resist.

But I like your idea of...

```
filters/
├── files.go
└── nodes.go
```

...specifically because of that `nodes.go` file there which can provide some helpful filtering functions for getting the AST declarations/statement/etc that people will be interested in.